### ISSUE TITLE
Define the problem as multi classes

### ISSUE THREAD

**Issue by michaelshekasta** [mrjleo/boilernet#8]
Hi,

Thanks for the great work! It's amazing. As I saw it, your model solves a binary classification.
If I want to solve a multiclass problem, how would you solve it?
I think to change the last layer into:
`        model.add(tf.keras.layers.Dense(3, activation='softmax'))`

and the loss:
`        model.compile(loss='categorical_crossentropy', optimizer='adam')
`

Is it enough? Should I change the preprocessing (label representation)?

Regards,
Michael

**Comment by mrjleo** [comment]
Hi, yes that looks right to me. In addition you'd have to change the labels to one-hot vectors and adjust the metric computations.

Take this with a grain of salt as I am not too familiar with TF/keras these days.
